### PR TITLE
[Doc] Giant graph tutorial edit for grammar and style

### PR DESCRIPTION
**Pull request by john-andrilla**
## Description
Edit pass for readability. Bundled multiple commits in one PR.
@aaronmarkham Review and approval appreciated here.

### CODE REVIEW

File: tutorials/models/1_gnn/9_gat.py [hunk 1]
```diff
@@ -1,40 +1,37 @@
...
 <https://github.com/mufeili>`_, `Minjie Wang
 <https://jermainewang.github.io/>`_  `Zheng Zhang
 <https://shanghai.nyu.edu/academics/faculty/directory/zheng-zhang>`_
 
-From `Graph Convolutional Network (GCN) <https://arxiv.org/abs/1609.02907>`_,
-we learned that combining local graph structure and node-level features yields
-good performance on node classification task. However, the way GCN aggregates
-is structure-dependent, which may hurt its generalizability.
+In this tutorial, you learn about a graph attention network (GAT) and how it can be 
+implemented in MXNet-DGL. You can also learn to visualize and understand what the attention
```
Comment (BarclayII) [inline tutorials/models/1_gnn/9_gat.py]: The code is implemented in PyTorch rather than MXNet.

Comment (john-andrilla) [inline tutorials/models/1_gnn/9_gat.py]:
```suggestion
implemented in PyTorch. You can also learn to visualize and understand what the attention 
```

Comment (john-andrilla) [reply tutorials/models/1_gnn/9_gat.py]: Thanks for the review. Updated the commit.

File: tutorials/models/1_gnn/9_gat.py [hunk 2]
```diff
@@ -77,30 +74,29 @@
...
 # In its essence, GAT is just a different aggregation function with attention
 # over features of neighbors, instead of a simple mean aggregation.
 #
 # GAT in DGL
 # ----------
 #
-# Let's first have an overall impression about how a ``GATLayer`` module is
-# implemented in DGL. Don't worry, we will break down the four equations above
-# one-by-one.
+# To begin, you can see an overall impression about how a ``GATLayer`` module is
```
Comment (aaronmarkham) [inline tutorials/models/1_gnn/9_gat.py]: I usually "get" an impression. Synesthesia?

Comment (john-andrilla) [inline tutorials/models/1_gnn/9_gat.py]:
```suggestion
# To begin, you can get an overall impression about how a ``GATLayer`` module is
```

Comment (john-andrilla) [inline tutorials/models/1_gnn/9_gat.py]:
```suggestion
# To begin, you can get an overall impression about how a ``GATLayer`` module is
```

**Comment by jermainewang** [comment]
> Looks good... had one word choice comment.
> 
> But also, maybe ping OP on why the readme is .txt instead of .rst? It would render better on github...

It's a convention of [sphinx-gallery](https://sphinx-gallery.github.io/stable/index.html). Quite strange to me as well.

**Pull request merged**